README.md

```diff
@@ -65,7 +65,7 @@ state files).
 [instance profiles]: http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html
 [aws-account-wide-terraform]: https://github.gds/government-paas/aws-account-wide-terraform
 
-* Declare you environment name using the variable DEPLOY_ENV. It must be 18 characters maximum and contain only alphanumeric characters and hyphens.
+* Declare your environment name using the variable DEPLOY_ENV. It must be 15 characters maximum and contain only alphanumeric characters and hyphens.
```

Comment:
We did think about this, and decided to go the easy route on the grounds that it's easy to assume the code is correct and to fix the docs if they ever get out of date.
I'd prefer to remove the 2 sources of truth for the value, but I wasn't certain that the makefile is the only user of the variable.
Happy to update the readme again, if you have a better steer?